src/Languages/Core/Impl/Utility/ListHelpers.cs

```diff
@@ -6,7 +6,7 @@
 
 namespace Microsoft.Languages.Core.Utility {
     public static class ListHelpers {
-        public static void RemoveDuplicates<T>(this List<T> list) where T : IComparable<T> {
+        public static void RemoveDuplicates<T>(this List<T> list, Comparison<T> comparison = null) where T : IComparable<T> {
```

Comment:
So the overload with `Comparison<T> comparison` can be called from the original method:
```{csharp}
public static void RemoveDuplicates<T>(this List<T> list) where T : IComparable<T> => list.RemoveDuplicates((i1, i2) => i1.CompareTo(i2));
```

---
In reply to: [80277594](https://github.com/Microsoft/RTVS/pull/2393#discussion_r80277594) [](ancestors = 80277594)